### PR TITLE
narwhal: enable keepalives on quic connections

### PR DESCRIPTION
**Pull request by bmwill**

### CODE REVIEW

File: narwhal/primary/src/primary.rs [hunk 1]
```diff
@@ -251,9 +251,19 @@ impl Primary {
             )))
             .into_inner();
 
+        let anemo_config = {
+            let mut quic_config = anemo::QuicConfig::default();
+            // Enable keep alives every 1s
```
Comment (arun-koshy) [inline narwhal/primary/src/primary.rs:256]: every 5s?

Comment (aschran) [inline narwhal/primary/src/primary.rs:256]: 1s or 5s?

File: narwhal/worker/src/worker.rs [hunk 2]
```diff
@@ -157,9 +157,20 @@ impl Worker {
                 outbound_network_metrics,
             )))
             .into_inner();
+
+        let anemo_config = {
+            let mut quic_config = anemo::QuicConfig::default();
+            // Enable keep alives every 1s
```
Comment (arun-koshy) [inline narwhal/worker/src/worker.rs:163]: 5 seconds here as well

**Review by arun-koshy — APPROVED** [review]

File: narwhal/worker/src/worker.rs (hunk 2 above)
Comment (aschran) [inline narwhal/worker/src/worker.rs:163]: same

**Review by aschran — APPROVED** [review]

**Comment by bmwill** [comment]
Thanks @arun-koshy and @aschran for catching the mismatch in the comments!

**Pull request merged**